.github/workflows/Android-CI.yml

```diff
@@ -0,0 +1,52 @@
+name: Android-CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
this limits the run to only pull requests made to master I would prefer this on every one